multimodal/src/autogluon/multimodal/configs/model/fusion_mlp_image_text_tabular.yaml

```diff
@@ -251,3 +215,26 @@ model:
     data_types:
     additive_attention: False # Whether to use lightweight additive attention, can be True, False or "auto"
     share_qv_weights: False # Whether to share weight for query and value, can be True, False or "auto"
+  ft_transformer:
+    data_types:
+      - "categorical"
+      - "numerical"
+    embedding_arch:
```

Comment:
Looks like the default value of `embedding_arch ` changes from `linear` and `relu` to `linear`.

Reply:
Default should be `linear`, which is the same as ft_transformer. The other `embedding_arch` are different variants.